src/DynamoCore/Search/SearchDictionary.cs

```diff
@@ -14,6 +14,17 @@ public class SearchDictionary<V>
     {
         protected readonly Dictionary<V, Dictionary<string, double>> entryDictionary =
             new Dictionary<V, Dictionary<string, double>>();
+        
+        private bool experimentalSearch = false;
+
+        /// <summary>
+        ///     Indicates whether experimental search mode is turned on.
+        /// </summary>
+        internal bool ExperimentalSearch
+        {
+            get { return experimentalSearch; }
+            set { experimentalSearch = value;  }
+        }
```

Comment:
When implementing my own debug mode VS suggested;
internal bool ExperimentalSearch { get; set; } = false;

Reply:
@ColinDayOrg Yep, that's cleaner. Fixed.